src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,20 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+
+        Truck truck = new Truck();
```

Comment:
be attentive with [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop)